extensions/vscode/src/views/homeView.ts

```diff
@@ -1144,7 +1146,7 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
 
   public refreshContentRecords = async () => {
     await this.refreshContentRecordData();
-    this.updateWebViewViewContentRecords(this.getSelectionState());
+    this.updateWebViewViewContentRecords();
```

Comment:
We don't want to send the selection here, instead we only want to update the data.

Reply:
`refreshContentRecords` happens based on file watchers. We don't want to send over the selection with this we just want the content records' data to be updated.

Part of the issue here was that the create was happening, and in between the logic where we capture the deployment created the file was made by the backend which triggered the file watchers.

This led to the Deployment getting selected at the end of the create logic, and at the same time the new data being sent to the webview with the old `this.getSelectionState()`.

If we limit the watchers to only updating data we avoid race conditions with other logic manipulating files.